src/d3a/d3a_core/redis_connections/redis_communication.py

```diff
@@ -35,12 +37,21 @@
 RESULTS_CHANNEL = "d3a-results"
 ZIP_RESULTS_CHANNEL = "d3a-zip-results"
 ZIP_RESULTS_KEY = "d3a-zip-results-key/"
+HEARTBEAT_CHANNEL = "d3a-heartbeat"
```

Comment:
This is not going to work with multiple simulations. The simulation_id needs to be part of the channel name

Reply:
`simulation_id` is sent as part of published message.

Reply:
Understood. Then all is good, ignore my comment.